product_docs/docs/edbcloud/beta/administering_cluster/01a_superset_access.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Managing Superset Access
+---
+
+You control access to Superset data sources by Superset roles and permissions. Learn more about Superset roles, users, and permissions management in [Superset Security](https://superset.apache.org/docs/security). To assign Superset permissions to the EDB Cloud user role, see [Changing Role Permissions](01_portal_access/#changing-role-permissions).
+
+EDB Cloud users can be granted the following special permissions:
+
+* superset:admin
+* superset:alpha
+* superset:gamma (granted by default)
+
+These EDB Cloud permissions translate to Superset roles as follows:
+
+
+| User Superset Role  | Permissions Included in EDB Cloud User Role                                         | Superset Privileges                                                                    |
```

Comment:
What does "Permissions Included in EDB Cloud User Role" mean?

Reply:
@vishalsawale9 , I need your help figuring out how to better title this column.

Reply:
See my latest commit for my attempt at clarifying this topic.

Reply:
We could use a title like "EDB Cloud role mapped for superset" or something like this. 

We just need to educate users about the mapping of roles between EDB Cloud and superset. So we should be little careful here and use the same phrase that we used in "Manage Portal Access" page for our RBAC.